diff --git a/configure b/configure
index 21657b0..2d509be 100755
--- a/configure
+++ b/configure
@@ -1550,8 +1550,13 @@ do
 
         msg "configuring LLVM for $gnu_t"
 
-        LLVM_TARGETS="--enable-targets=x86,x86_64,arm,aarch64,mips,powerpc"
-        LLVM_BUILD="--build=$gnu_t"
+        if [ -z $LLVM_TARGETS ];
+	then
+		LLVM_TARGETS="--enable-targets=x86,x86_64,arm,aarch64,mips,powerpc"
+        else
+		LLVM_TARGETS="--enable-targets=$LLVM_TARGETS"
+	fi
+	LLVM_BUILD="--build=$gnu_t"
         LLVM_HOST="--host=$gnu_t"
         LLVM_TARGET="--target=$gnu_t"
 
